backtrader/brokers/ccxtbroker.py

```diff
@@ -102,7 +102,7 @@ def sell(self, owner, data, size, price=None, plimit=None,
         return self._submit(owner, data, exectype, 'sell', size, price, kwargs)
 
     def cancel(self, order):
-        return self.store.cancel_order(order['id'])
+        return self.store.cancel_order(order.ccxt_order['id'])
```

Comment:
Hi,

Thank you for this PR. The change makes sense.

Just one thing - it would be better to hide ccxt-related internals in the CCXTStore code, i.e. pass order to the store as is and get its id inside the store. Something like this:
```
        return self.store.cancel_order(order.ccxt_order)
```

Reply:
in general I agree.

but because CCXTOrder is on the CCXTBroker.py it makes the code a bit more complex to understand. 

the store cancel_order only need the id and doesn't have access to CCXTOrder class.
and changing the signature of  'def cancel_order(self, order_id):'    --->   def cancel_order(self, ccxt_order): 
makes it unclear to what ccxt_order is

Reply:
> but because CCXTOrder is on the CCXTBroker.py it makes the code a bit more complex to understand.

Good point. I'm planning to move it to the CCXTStore as well to avoid using any ccxt internals in the broker class.

 > changing the signature of 'def cancel_order(self, order_id):' ---> def cancel_order(self, ccxt_order): makes it unclear to what ccxt_order is

Well, I tend to disagree here. From my point of view order_id is much harder to understand as it could be either backtrader or ccxt order id.

Reply:
I agree.

after moving the CCXTOrder it will make a lot more sense also to use ccxt_order.